File: Basics/Palindrome/palind.py
def is_palindrome(s):
    s = ''.join(c.lower() for c in s if c.isalnum())
    return s == s[::-1]

# Example usage:
string_to_check = "A man, a plan, a canal: Panama"
result = is_palindrome(string_to_check)
print(result)

File: Basics/Palindrome/palind.js
function isPalindrome(s) {
    s = s.replace(/[^a-zA-Z0-9]/g, '').toLowerCase();
    console.log(s)
    return s === s.split('').reverse().join('');
}

// Example usage:
let stringToCheck = "A man, a plan, a canal: Panama";
let result = isPalindrome(stringToCheck);
console.log(result);
